group_vars/all

```diff
@@ -25,3 +25,10 @@ validate_certs: "yes"
 # env:
 # service_vip:
 # control_interface:
+
+# should mons also run osds
+mon_osd_colocation_allowed: "true"
+
+# OSDs could be run without netplugin on them
+# osd_contiv_network_include controls this behaviour
+osd_contiv_network_include: "true"
```

Comment:
just to add if we really need negation it might be better achieved using logical operator so a node with no storage capability can matched using this condition `when: ! host-capability | match(*storage*)`